scvi/external/cell2location/_base.py

```diff
@@ -0,0 +1,1060 @@
+from datetime import date
+from typing import Optional
```

Comment:
run import sort `isort`

Reply:
what do you mean? is this a command-line app?

Reply:
yes, if you install `pre-commit` as described in our [contributing guide](https://docs.scvi-tools.org/en/stable/contributing/index.html), it won't let you push a commit without it passing black formatting, flake8, and isort (and will fix it for you)